exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/VideoInviteActivity.java

```diff
@@ -479,17 +482,35 @@ private void createLocalTracks() {
         localAudioTrack = LocalAudioTrack.create(this, true);
 
         // Share your camera
-        cameraCapturer = new CameraCapturer(this, getAvailableCameraSource());
+        cameraCapturer = new CameraCapturer(this, getFrontCameraId());
         localVideoTrack = LocalVideoTrack.create(this, true, cameraCapturer);
         primaryVideoView.setMirror(true);
         localVideoTrack.addSink(primaryVideoView);
         localVideoView = primaryVideoView;
     }
 
-    private CameraSource getAvailableCameraSource() {
-        return (CameraCapturer.isSourceAvailable(CameraSource.FRONT_CAMERA)) ?
-                (CameraSource.FRONT_CAMERA) :
-                (CameraSource.BACK_CAMERA);
+    private String getFrontCameraId() {
+        if (frontCameraId != null) {
```

Comment:
This null check should be removed right? It looks like the code within the if statement will never be executed since it first gets assigned on line 496

Reply:
Good catch! The check is needed because this method acts as a "lazy getter", but there was a bug in the check itself. It should be `==`.